doc/users/whats_new.rst

```diff
@@ -26,6 +26,16 @@ new in matplotlib-1.3
 New plotting features
 ---------------------
 
+Easy color shading
+``````````````````
+Often we would like the ability add subtle color variation to our plots,
+especially when similar element are plotted in close proximity to one another.
+:func:`matplotlib.colors.shade_color` can be used to take an existing color and
+slightly darken it, by giving a negative percentage, or to slightly lighten it,
+by giving a positive percentage.
+
+.. plot:: mpl_examples/color/color_shade_demo.py
+
```

Comment:
I see why this wants a rebase... Given that version 1.3 is listed at the top of your what's new file, I think you need to update your fork of matplotlib. Once you got your local repo all up to date, then rebase this branch off of the updated master. In that process, you should also move this entry to its own file, which we do now to avoid merge conflicts. We automatically assemble all of the what's new entry prior to release.